docs/source/ecosystem.rst

```diff
@@ -34,6 +34,9 @@ DataFrame
   GPU-enabled dataframes which can be used as partitions in Dask Dataframes.
 - `dask-geopandas <https://github.com/geopandas/dask-geopandas>`_: Early-stage subproject of
   geopandas, enabling parallelization of geopandas dataframes.
+- `Modin <https://modin.readthedocs.io/en/latest/development/using_pandas_on_dask.html>`_: Scale your
+  pandas workflows by changing one line of code. Modin transparently distributes the data and computation
+  using different execution engines such as Ray, Dask or MPI.
```

Comment:
```suggestion
- `Modin <https://github.com/modin-project/modin>`_: Scale your pandas workflows
  by changing one line of code. Modin transparently distributes data and computation
  using different execution engines such as
  `Ray <https://modin.readthedocs.io/en/latest/development/using_pandas_on_ray.html>`_,
  `Dask <https://modin.readthedocs.io/en/latest/development/using_pandas_on_dask.html>`_, or
  `MPI <https://modin.readthedocs.io/en/latest/development/using_pandas_on_mpi.html>`_,.
```

Reply:
@martindurant, @mrocklin, could you take a look at this PR? Can we merge it in the current state or with the suggestion above? Maybe you also have comments.

Reply:
@mrocklin, thank you for having a look at the changes. To address your comment another suggestion would be.

```
Modin is a drop-in replacement for pandas. It transperantly distributes data and computation using an execution engine, one of which is Dask.
```

Reply:
cc @fjetter

Reply:
General comment: this whole page has seen far to little updating considering the rapidly changing ecosystem.

My suggestion for the line:
> Modin [link] aims for 100% API compatibility with pandas, but distributes data and computation using one of dask, ray or mpi [further links not needed here]

(note that I cannot merge this PR)

Reply:
Your suggestion looks good to me too. Let's wait for comments from other maintainers.